Workflow/Sources/SubtreeManager.swift

```diff
@@ -82,15 +82,19 @@ extension WorkflowNode {
             childWorkflows = context.usedChildWorkflows
             sideEffectLifetimes = context.usedSideEffectLifetimes
 
-            /// Captured the reusable sinks from this render pass.
+            /// Capture the reusable sinks from this render pass.
             previousSinks = context.sinkStore.usedSinks
 
             /// Capture all the pipes to be enabled after render completes.
             eventPipes = context.eventPipes
-            eventPipes.append(contentsOf: context.sinkStore.eventPipes)
+            for (_, sink) in context.sinkStore.usedSinks {
```

Comment:
previously this was going through a call to `map` which created an intermediate buffer of the mapped elements. we don't need them to be stored separately so we can just do the iteration inline to avoid that